docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>I think the comment is not really necessary + would move the geometry creation to the cell below.</p><p>Would also prefer to have 1e-3 instead of 0.001.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>